pylint_odoo/checkers/modules_odoo.py

```diff
@@ -829,19 +829,17 @@ def _get_module_files(self):
 
     def _check_file_not_used(self):
         """Check if a file is not used from manifest"""
-        self.msg_args = []
         module_files = set(self._get_module_files())
         referenced_files = set(self._get_manifest_referenced_files()).union(
             set(self._get_xml_referenced_files())
         )
-        for no_referenced_file in (module_files - referenced_files):
-            if (not no_referenced_file.startswith('static/') and
-                not (no_referenced_file.startswith('test/') or
-                     no_referenced_file.startswith('tests/'))):
-                self.msg_args.append((no_referenced_file,))
-        if self.msg_args:
-            return False
-        return True
+        excluded_dirs = ['static', 'test', 'tests', 'migrations']
+        no_referenced_files = [
+            file for file in (module_files - referenced_files)
```

Comment:
file is a reserverd keyword

Reply:
use a normal for to use a break early
or you can use sets too here?

Reply:
- Done, changed the "file" variable. Thanks, good catch!
- I think an early break wouldn't be convenient in this case. Even though the method returns a boolean, it also stores all failing files and, if we break the loop, it could lead to non-detected failing files.